### PR TITLE
Add (README.md): Add instructions for setting up systemd service

### PR DESCRIPTION
**Pull request by josephmturner**
From Adam's work [here](https://todo.sr.ht/~ushin/ushin/13#event-229700).

### CODE REVIEW

**Comment by josephmturner** [comment]
Resolves #7. (except for downloading the binary)

File: README.md [hunk 1]
```diff
@@ -50,3 +50,29 @@ Hyper-gateway uses the [pkg](https://github.com/vercel/pkg) module to compile th
...
+
+# This will create the service file.
+cat << EOF > ~/.local/share/systemd/user/hyper-gateway.service
+[Unit]
+Description=hypercore-protocol gateway daemon (for hyperdrive)
+Documentation=https://github.com/RangerMauve/hyper-gateway
+
+[Service]
+Type=exec
+ExecStart=%h/.local/bin/hyper-gateway --writable true --silent true run
```
Comment (RangerMauve) [inline README.md:86]: Would it be possible to have more docs added to this to explain how to get `hyper-gateway` into that location? Or what to do if you don't already have it there.

File: README.md [hunk 2]
```diff
@@ -50,3 +50,29 @@ Hyper-gateway uses the [pkg](https://github.com/vercel/pkg) module to compile th
 - `npm install`
 - `npm run build`
 - Look in the `dist` folder for the platform you want.
+
+## FAQ
+
+### How to run hyper-gateway as a background process?
```
Comment (RangerMauve) [inline README.md]:
```suggestion
### How can I run hyper-gateway as a background process? (On Linux)
```

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Good start, I think without the info on the binary it might confuse folks though.

File: README.md (hunk 1 above)
Comment (josephmturner) [reply README.md:86]: Please check latest commit :)

**Review by RangerMauve — APPROVED** [review]
This is perfect, thank you!

**Pull request merged**